lick/drum/AkaiMpc60.ck

```diff
@@ -20,87 +20,90 @@
 
 */
 
+@import "../sample/Sample"
+@import "../sample/Reconnect"
+
 class ClosedHat extends Sample
 {
     {
-        "samples/AkaiMpc60/ClosedHat.wav" => buf.read;
+        me.dir() + "../../samples/AkaiMpc60/ClosedHat.wav" => buf.read;
```

Comment:
Can samples be part of a package?  If not, am not sure how this relative directory lookup would work outside of this source tree

Reply:
Yes! Any arbitrary files can be included in the package. The package version of this would just preserve the directory structure of the repo. How are these samplers typically used? Do people mostly copy and pasted the needed parts of LiCK?